### ISSUE TITLE
Is this repo still actively maintained?

### ISSUE THREAD

**Issue by david-fortini** [inim-repl/INim#138]
If not, are there alternatives?

**Comment by lf-araujo** [comment]
Not the dev, but this is pretty much complete. Inim is a bit slow, but as nim approaches v2, speed should trickle down to Inim.

I do have problems with indentation, but I found out about a neovim plugin called Codi, and implemented inim on it. So I can have a scratchpad like development, without the indentation problems. It works well in my opinion, though slightly slow compared with python or R.

**Comment by niontrix** [comment]
@lf-araujo could you tell me how you implemented the interaction with Codi? I'm trying to build an inferior mode for Emacs, but I currently am stuck on being able to properly get multi-line inputs to work. Maybe this could give me some hints.

**Comment by lf-araujo** [comment]
Yes, sure. The implementation is public and can be found here: https://github.com/metakirby5/codi.vim/commit/83b9859aaf8066d95892e01eb9c01571a4b325dd

After much testing, this approach clearly is too slow for very long scripts. 

My intention is to go back to this and try implementing to nlvm: https://forum.nim-lang.org/t/10697

**Comment by niontrix** [comment]
Thanks for the links! `nlvm` looks really interesting. This is exciting :smile:
